modules/auth0/app/README.md

```diff
@@ -0,0 +1,135 @@
+# Component: `auth0/app`
+
+Auth0 Application component. Auth0 is a third-party service that provides authentication and authorization as a service.
```

Comment:
nit: add link to Auth0's landing page or documentation landing page. Not a must but just nice to have in the README IMO.

Reply:
added hyperlinks